borsh/src/lib.rs

```diff
@@ -41,7 +41,6 @@ pub mod maybestd {
 
     pub mod collections {
         pub use alloc::collections::{BTreeMap, BTreeSet, BinaryHeap, LinkedList, VecDeque};
-        pub use hashbrown::*;
```

Comment:
We cannot do this just like this. `HashMap` and `HashSet` are still used when `std` feature is disabled. Unfortunately, tests did not catch this (pull in #144 for the fix to see the compilation errors if you remove `hashbrown` just like this):

<img width="1136" alt="image" src="https://github.com/near/borsh-rs/assets/304265/ced50554-ecef-4afd-9fee-770464d90c97">

Please, revert this change (including the change in Cargo.toml)